db/schema.rb

```diff
@@ -57,6 +56,7 @@
     t.string   "media_url"
     t.boolean  "visible_content"
     t.boolean  "visible_image"
+    t.boolean  "media_landscape", default: false
```

Comment:
I'm assuming we stored this because you want me to have some control somewhere about whether a media image should be larger landscape wise or not?

Can you open up an issue for all the things you want backend to track about an incoming image? I'm guessing if we get an image from twitter or somewhere, we should do some image processing to convert it into whatever you need?  Or can we just use image width/length?  Regardless, let's document this somewhere in an open issue so someone can work on it.